pdns/pdnsutil.cc

```diff
@@ -3854,7 +3854,7 @@ try
     }
     DNSName zone(cmds.at(1));
     string kind = cmds.at(2);
-    static vector<string> multiMetaWhitelist = {"ALLOW-AXFR-FROM", "ALLOW-DNSUPDATE-FROM",
+    const static std::array<string, 7> multiMetaWhitelist = {"ALLOW-AXFR-FROM", "ALLOW-DNSUPDATE-FROM",
       "ALSO-NOTIFY", "TSIG-ALLOW-AXFR", "TSIG-ALLOW-DNSUPDATE", "GSS-ALLOW-AXFR-PRINCIPAL",
       "PUBLISH-CDS"};
```

Comment:
Same maybe an Array of `string_view` is better but preferred to leave a question can move into an other PR!

Reply:
Some question here: what's the reason you suggest string_view instead of just string?